File: Examples/Ruby/Workbook/post_workbook_get_smart_marker_result.rb
require 'aspose_cells_cloud'

class Workbook

  include AsposeCellsCloud
  include AsposeStorageCloud

  def initialize
    #Get App key and App SID from https://cloud.aspose.com
    AsposeApp.app_key_and_sid("APP_KEY", "APP_SID")
    @cells_api = CellsApi.new  
  end

  def upload_file(file_name)
    @storage_api = StorageApi.new 
    response = @storage_api.put_create(file_name, File.open("../data/" << file_name,"r") { |io| io.read } )
  end

  def post_workbook_get_smart_marker_result
    file_name = "Sample_SmartMarker.xlsx"
    upload_file(file_name)
    data_file = "Sample_SmartMarker_Data.xml"

    response = @cells_api.post_workbook_get_smart_marker_result(file_name, File.open("../data/" << data_file,"r") { |io| io.read })
  end

end

workbook = Workbook.new()
puts workbook.post_workbook_get_smart_marker_result

File: Examples/PHP/Workbook/PostWorkbookGetSmartMarkerResult.php
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\Cells\CellsApi;
use Aspose\Cells\AsposeApp;

class Workbook {

	public $cells;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->cells = new CellsApi();
	}
	
	public function postWorkbookGetSmartMarkerResult() {
    	// Upload file to Aspose Cloud Storage
		$fileName = "Sample_SmartMarker.xlsx";
		Utils::uploadFile($fileName);

        $file = realpath(__DIR__ . '/../../..') . '/Data/Sample_SmartMarker_Data.xml';
        $result = $this->cells->PostWorkbookGetSmartMarkerResult($name=$fileName, $xmlFile = null, $storage = null, $folder = null, $outPath = "smartmarker1.xlsx", $file);
        print_r($result);
	}	
}

$workbook = new Workbook();
$workbook->postWorkbookGetSmartMarkerResult();

?>